lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -517,7 +524,11 @@ def rescore_problem_module_state(xmodule_instance_args, module_descriptor, stude
             TASK_LOG.warning(msg)
             return UPDATE_STATUS_FAILED
 
-        if not hasattr(instance, 'rescore_problem'):
+        for method in ['rescore', 'rescore_problem']:
```

Comment:
Add a TODO statement with the ticket number to remember to update this code once CAPA is updated.